src/hooks/useCreateBlog.ts

```diff
@@ -0,0 +1,54 @@
+import { axiosInstance } from "@/lib/axios";
+import { useMutation } from "@tanstack/react-query";
+import { AxiosError } from "axios";
+import { useRouter } from "next/navigation";
+
+interface Payload {
+  author: string;
+  category: string;
+  content: string;
+  description: string;
+  thumbnail: File;
+  title: string;
+}
+
+interface ResponseThumbnail {
+  fileURL: string;
+  filePath: string;
+}
```

Comment:
mungkin ini bisa di pindahkan ke folder type
```suggestion
interface Payload {
  author: string;
  category: string;
  content: string;
  description: string;
  thumbnail: File;
  title: string;
}

interface ResponseThumbnail {
  fileURL: string;
  filePath: string;
}
```